tests/rbd/rbd_utils.py

```diff
@@ -481,6 +481,8 @@ def image_resize(self, pool_name, image_name, size, **kw):
             for passphrase in kw.get("passphrase"):
                 for key, value in passphrase.items():
```

Comment:
[optional]
Sorry for adding this now

Can we add info on passphrase in method doc ?
Also, if you can give me context on relationship with "encryption-passphrase-file" from method doc, that would be helpful - is it specific to encrypted image ?

Reply:
encryption-passphrase-file is a parameter passed to rbd resize command when the image in consideration is encrypted.
What more context do you want Vasi? Could you please elaborate?

I have also added some method doc with example to image_resize method. So please let me know if you need any more info.

Reply:
Got it
Thank you